src/nvim/buffer.c

```diff
@@ -3961,30 +3961,6 @@ char *buf_spname(buf_T *buf)
   return NULL;
 }
 
-/// Find a window for buffer "buf".
-/// If found true is returned and "wp" and "tp" are set to
-/// the window and tabpage.
-/// If not found, false is returned.
-///
-/// @param       buf  buffer to find a window for
-/// @param[out]  wp   stores the found window
-/// @param[out]  tp   stores the found tabpage
-///
-/// @return  true if a window was found for the buffer.
-bool find_win_for_buf(buf_T *buf, win_T **wp, tabpage_T **tp)
```

Comment:
This function is no longer used in Nvim, probably because all its usages have been replaced by `aucmd_prepbuf()`.